classes/challenges/DoorPassword.js

```
@@ -0,0 +1,34 @@
+
+  static frontDoorPassword(word) {
+    return this.capitalize(word);
+  }
+
+  static backDoorResponse(line) {
+    line = line.trim();
+    return line.length > 0 ? line[line.length - 1] : "";
+  }
+
+  static backDoorPassword(word) {
+    return this.capitalize(word) + ", please";
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)